djstripe/models/billing.py

```diff
@@ -1213,6 +1215,136 @@ def update_name(self):
         self.save()
 
 
+class Price(StripeModel):
+    """
+    Prices define the unit cost, currency, and (optional) billing cycle for
+    both recurring and one-time purchases of products.
+
+    Prices replace the old "Plan" model.
```

Comment:
It is a bit confusing to use 'model' here - both models exist, and will be supported in dj-stripe (i hope) until Stripe kills Plans, so there is no 'replace' happening.

"Prices are the Stripe component that will replace Plans."?  or "... that provide a richer model than Plans."

"As of 2020, Prices are able to be used wherever Plans were used."  (im not sure about exact details here, but that seems to be the situation)